lib/services/local_sync_service.dart

```diff
@@ -330,19 +335,15 @@ class LocalSyncService {
     List<File> files,
     Set<String> existingLocalFileIDs,
   ) async {
-    final updatedFiles = files
-        .where((file) => existingLocalFileIDs.contains(file.localID))
+    final List<String> updatedLocalIDs = files
+        .where(
+          (file) =>
+              file.localID != null &&
+              existingLocalFileIDs.contains(file.localID),
+        )
+        .map((e) => e.localID!)
         .toList();
-    if (updatedFiles.isNotEmpty) {
-      _logger.info(
-        updatedFiles.length.toString() + " local files were updated.",
-      );
```

Comment:
Why remove this logging?

Reply:
Enhanced the logging inside `FileUpdationDB.instance.insertMultiple`